components/encryption/export/examples/ecli.rs

```diff
@@ -61,6 +62,27 @@ struct KmsCommand {
     /// Remote region.
     #[structopt(long)]
     region: Option<String>,
+    /// Azure KMS
+    #[structopt(subcommand)]
+    azure: Option<KmsSubCommandAzure>,
```

Comment:
How about renaming `Command::Kms` to `Command::Aws` and adding `Command::Azure(KmsSubCommandAzure)`?